app/src/main/java/com/github/epfl/meili/util/RotationGestureDetector.kt

```diff
@@ -5,7 +5,7 @@ import kotlin.math.atan2
 
 // Reference used : https://stackoverflow.com/questions/10682019/android-two-finger-rotation
 /** Detects two finger rotation motions */
-class RotationGestureDetector(// Object which can be rotated using two fingers
+open class RotationGestureDetector(// Object which can be rotated using two fingers
```

Comment:
Why is this class being made open?

Reply:
It's a leftover from when I was testing the class, I will remove it.